docs/vc_patch.md

```diff
@@ -0,0 +1,113 @@
+# Nintendo 2DS/3DS Virtual Console Patch
+
+The Nintendo Virtual Console for the 2DS and 3DS can play Game Boy Color ROMs bundled into a `.cia` (CTR Importable Archive) file. Creating a `.cia` file for a `.gbc` ROM file is outside the scope of this project.
+
+An optional file bundled in the `.cia` archive is a `.patch` file with the same name as the `.gbc` file, used to determine when and under what conditions to enable certain Virtual Console functionalities, such as the Wireless Link capabilities. It also allows arbitrary content in the ROM to be modified, such as for disabling the Game Boy Printer features of Pokémon Crystal.
```

Comment:
```suggestion
The Nintendo Virtual Console is an emulator for the Nintendo 2DS and 3DS consoles that runs (among others) Game Boy Color titles, while applying enhancements to some titles.

A `.patch` file is shipped within the emulator, which is used to determine when and under what conditions to enable certain enhancements, such as the Wireless Link capabilities used to replicate the original Link Cable functionality. It also allows arbitrary content in the ROM to be modified, such as for disabling the Game Boy Printer features of Pokémon Crystal.
```

Or something like this, we don't really need to explain neither the distribution format (`.cia`, this isn't even how it's stored on the nintendo servers, it's mainly used by developers), nor how the internal structure of the file looks (who cares what the files are called).

Reply:
> A `.patch` file is shipped within the emulator

It's within the .cia file though, not the emulator, right? That's the point of being able to generate valid .patch files even if you've changed the values of `FISSURE`, `wMenuJoypad`, etc: you can bundle your hack's .gbc and .patch in a .cia file and have the VC enhancements for Crystal still work. I think that much detail is needed in the explanation so people know what to do with the `make` output.

Reply:
I've made some edits based on your comment.

Reply:
> It's within the .cia file though, not the emulator, right?

Yeah sorry, I meant "shipped with".